src/styles/components/_home-search-box.scss

```diff
@@ -1,18 +1,66 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  display: flex;
+  width: 80%;
+
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px) 0px box-shadow-opacity(0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
+
+  &__input {
+    width: 100%;
+    box-sizing: border-box;
+    padding: rem(5px 10px 5px 10px);
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+    text-align: left;
+
+    @include respond-above('sm') {
+      padding: rem(10px 20px 10px 20px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      padding: rem(20px 40px 20px 40px);
+      font-size: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(35px 60px 35px 60px);
+      font-size: rem(24px);
+    }
+  }
+
+  &__input:focus {
+    outline: none;
+  }
 
   &__button {
-    @include button-darken-effect($picton-blue);
+    width: 10%;
+    padding: 0;
+    background-color: $light;
+    border: none;
+    background: {
+      image: url('/icons/search-icon.svg');
+      repeat: no-repeat;
+      size: 70%;
+      position: left;
+    }
 
-    text-decoration: none;
-    padding: rem(5px 10px);
-    border-radius: rem(3px);
-    background-color: $picton-blue;
+    @include respond-above('md') {
+      width: 7%;
```

Comment:
7% makes the button to small, just go for 10%. Also the button is not in a nice ratio.

![Screen Shot 2564-03-01 at 01 04 15](https://user-images.githubusercontent.com/32289827/109428564-200efb80-7a2a-11eb-9c5c-5554740e9851.png)